x-pack/plugin/sql/qa/src/main/resources/examples/example.sql-spec

```diff
@@ -5,4 +5,28 @@ name
 // SQL query to be executed against H2 and ES
 SELECT COUNT(*) FROM "emp";
 
-// repeat the above
\ No newline at end of file
+// repeat the above
+
+
+// Test failure example:
+// org.elasticsearch.xpack.sql.qa.single_node.JdbcSqlSpecIT > test {agg.testGroupByOnText} FAILED
```

Comment:
This stacktrace is not useful, one with the .sql resource in it is more common (similar to the csv one).